packages/sdk/src/client.ts

```diff
@@ -506,6 +631,44 @@ export class AcrossClient {
     }
   }
 
+  /**
+   * Get a swap quote for a given set of parameters. See {@link getSwapQuote}.
+   * @param params - See {@link GetSwapQuoteParams}.
+   * @returns See {@link SwapApprovalApiResponse}.
+   */
+  async getSwapQuote(
+    params: MakeOptional<GetSwapQuoteParams, "logger" | "apiUrl">,
+  ): Promise<SwapApprovalApiResponse> {
+    try {
+      const quote = await getSwapQuote({
+        ...params,
```

Comment:
Should we force setting `skipOriginTxEstimation: true` here? So we can assume that if the API returns a simulation error, it is a fill simulation